posix.mak

```diff
@@ -725,16 +725,14 @@ else
  DMD_EXCLUDE += -e /scanmach/d -e /libmach/d
 endif
 
-# remove this after https://github.com/dlang/dmd/pull/7513 has been merged
-DMD_EXCLUDE_LATEST =
-ifneq (,$(wildcard $(DMD_DIR)/src/dmd/objc_glue_stubs.d))
-   DMD_EXCLUDE_LATEST += -e /objc_glue.d/d
-endif
-
 $G/docs-latest.json : ${DMD_LATEST} ${DMD_LATEST_DIR} \
 			${DRUNTIME_LATEST_DIR} ${PHOBOS_LATEST_FILES_GENERATED} | dpl-docs
+	# remove this after https://github.com/dlang/dmd/pull/7513 has been merged
+	if [ -f $(DMD_LATEST_DIR)/src/*/objc_glue_stubs.d ] ; then \
+	   DMD_EXCLUDE_LATEST="-e /objc_glue.d/d"; \
+	fi; \
 	find ${DMD_LATEST_DIR}/src -name '*.d' | \
-		sed -e /mscoff/d ${DMD_EXCLUDE_LATEST} ${DMD_EXCLUDE} \
+		sed -e /mscoff/d $${DMD_EXCLUDE_LATEST} ${DMD_EXCLUDE} \
```

Comment:
Should there be two `$`?

Reply:
Yes it's now a shell variable and `$$` escapes the `$`.

Reply:
Ok.